redfish-core/lib/log_services.hpp

```diff
@@ -1795,6 +1830,13 @@ inline void requestRoutesDBusEventLogEntry(App& app)
                                 filePath = std::get_if<std::string>(
                                     &propertyMap.second);
                             }
+
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+                            else if (propertyMap.first == "ManagementSystemAck")
+                            {
+                                bool* managementsaptr =
+                                    std::get_if<bool>(&propertyMap.second);
+                                if (managementsaptr == nullptr)
```

Comment:
Looks like a bad merge resolve?

Reply:
yes, I am going to fix and test it on Hw and then upload again .